lib/DiscoveryStreamFeed.jsm

```diff
@@ -606,8 +615,10 @@ this.DiscoveryStreamFeed = class DiscoveryStreamFeed {
   async getComponentFeed(feedUrl, isStartup) {
     const cachedData = await this.cache.get() || {};
     const {feeds} = cachedData;
+
     let feed = feeds ? feeds[feedUrl] : null;
-    if (this.isExpired({cachedData, key: "feed", url: feedUrl, isStartup})) {
+    const isExpired = this.isExpired({cachedData, key: "feed", url: feedUrl, isStartup});
+    if (isExpired) {
```

Comment:
do we need const isExpired or is it for code readability?

Reply:
I think I did this because I was logging stuff while debugging, and needed to explicitly log that value, I jut left it because it seemed harmless. I can revert it.